packages/build/src/utils/semver.js

```diff
@@ -32,3 +32,9 @@ export const getMajorVersion = function (version) {
   const patchVersion = semver.patch(version)
   return `${majorVersion}.${minorVersion}.${patchVersion}`
 }
+
+export const isPrerelease = function (version) {
+  // Prerelease versions are excluded from ranges unless `includePrerelease` is
+  // set to `true`.
+  return !semver.satisfies(version, '>=0.0.0', { includePrerelease: false })
```

Comment:
Could you use `semver.prerelease()`?

Reply:
You prefer a simpler and more expressive method? FINE, whatever.

(This method is not in the docs, so I had to get creative. Thanks!)

Done in https://github.com/netlify/build/pull/5554/commits/adc65d4e47de834cb8c498412ec8f1bfef1a8ad7.